pages/add/add.jsx

```diff
@@ -1,12 +1,201 @@
+          </fieldset>
+          <label className="form-control-label">
+            Keywords to help with search by program, event, campaign, subject …
+            <input name="tags" type="text" placeholder="#mozfest  #code  #tool" className="form-control" />
+          </label>
+          <fieldset>
+            <legend>Check any Key Internet Issues that relate to your project.</legend>
+            <label className="form-control-label">
+              <input name="issues" type="checkbox" className="form-control" value="Online Privacy & Security" />Online Privacy & Security
+            </label>
+            <label className="form-control-label">
+              <input name="issues" type="checkbox" className="form-control" value="Open Innovation" />Open Innovation
+            </label>
+            <label className="form-control-label">
+              <input name="issues" type="checkbox" className="form-control" value="Decentralization" />Decentralization
+            </label>
+            <label className="form-control-label">
+              <input name="issues" type="checkbox" className="form-control" value="Web Literacy" />Web Literacy
+            </label>
+            <label className="form-control-label">
+              <input name="issues" type="checkbox" className="form-control" value="Digital Inclusion" />Digital Inclusion
+            </label>
+          </fieldset>
+
+
+          <label className="form-control-label">
+            Looking for support? Describe how people can do that.
+            <input name="get_involved" type="text" placeholder="Contribute to the code." className="form-control" />
+          </label>
+          <label className="form-control-label">
+            Link for people to get involved.
+            <input name="get_involved_url" type="url" placeholder="https://example.com" className="form-control" />
+          </label>
+          <fieldset>
+            <label className="form-control-label">
+              Upload a thumbnail image
+              <input name="thumbnail_url" type="url" placeholder="https://example.com" className="form-control" />
+            </label>
+            <p className="form-text"><em>1200px by 630px recommended image dimension</em></p>
+          </fieldset>
+          <button type="submit" className="btn btn-filled">Submit project</button>
+        </form>
+      </div>
+    );
+
     return (
       <div className="add-page">
-        <div id="add-project-form">
-          <iframe src="https://docs.google.com/forms/d/1CiNYA3gzHPd4HYrgQuQB2KM0TDpsbn6jbMcl-0_OjM4/viewform?embedded=true" width="100%" height="100%" frameBorder="0" marginHeight="0" marginWidth="0">Loading...</iframe>
-        </div>
+        {
+          this.state.loggedInStatusChecked ? ( UserData.userLoggedInStatus.get() ? ContentForAuthedUser : ContentForUnauthedUser )
```

Comment:
if you need nested ternaries, don't use ternaries. Instead write out the code in an easier to read and maintain format.